frontend/src/components/pages/CampCreation/ScheduleSessions/SessionSidePanel.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Box } from "@chakra-ui/react";
+import { CreateCampSession } from "../../../../types/CampsTypes";
+import AddSessionsForm from "./AddSessionsForm";
+import CurrentSessionsView from "./CurrentSessionsView";
+
+type SessionSidePanelProps = {
+  scheduledSessions: CreateCampSession[];
+  setScheduledSessions: (sessions: Array<CreateCampSession>) => void;
+};
+
+const SessionSidePanel: React.FC<SessionSidePanelProps> = ({
+  scheduledSessions,
+  setScheduledSessions,
+}: SessionSidePanelProps): React.ReactElement => {
+  const [showAddSessionView, setShowAddSessionView] = React.useState(false);
```

Comment:
Don't think it's worth the change, but why use this instead of an enum?

Reply:
Ah I didn't think very hard abt this - this just felt most intuitive for me - can do enum moving frwd